### PR TITLE
annotated stuff

### PR DESCRIPTION
**Pull request by Asergey91**

### CODE REVIEW

**Review by nvdk — APPROVED** [review]
looks good!

**Comment by nvdk** [comment]
@Asergey91 did you also create a migration for this in app-gelinkt-notuleren?

File: standaard-templates/besluit/nieuw/aanvullend-reglement-besluit.ttl [hunk 1]
```diff
@@ -42,7 +42,7 @@
...
      <li>wet <a class=\"annotation\" href=\"https://codex.vlaanderen.be/doc/document/1009628\" property=\"eli:cites\" typeof=\"eli:LegalExpression\">betreffende de politie over het wegverkeer (wegverkeerswet - Wet van 16 maart 1968)</a></li>
      <li>wegcode - Koninklijk Besluit <a class=\"annotation\" href=\"https://codex.vlaanderen.be/doc/document/1036242\" property=\"eli.cites\" typeof=\"eli:LegalExpression\">van 1 december 1975 houdende algemeen reglement op de politie van het wegverkeer en van het gebruik van de openbare weg.</a></li>
-     <li>code van de wegbeheerder - ministerieel besluit van 11 oktober 1976 houdende de minimumafmetingen en de bijzondere plaatsingsvoorwaarden van de verkeerstekens</li>
+     <li>code van de wegbeheerder - <a class=\"annotation\" href=\"https://codex.vlaanderen.be/doc/document/1035575\" property=\"eli.cites\" typeof=\"eli:LegalExpression\">ministerieel besluit van 11 oktober 1976 houdende de minimumafmetingen en de bijzondere plaatsingsvoorwaarden van de verkeerstekens</a></li>
```
Comment (abeforgit) [inline standaard-templates/besluit/nieuw/aanvullend-reglement-besluit.ttl]: please pay attention to correct rdfa syntax
I see it's also wrong in the line above
```suggestion
     <li>code van de wegbeheerder - <a class=\"annotation\" href=\"https://codex.vlaanderen.be/doc/document/1035575\" property=\"eli:cites\" typeof=\"eli:LegalExpression\">ministerieel besluit van 11 oktober 1976 houdende de minimumafmetingen en de bijzondere plaatsingsvoorwaarden van de verkeerstekens</a></li>
```

**Review by abeforgit — CHANGES_REQUESTED** [review]

File: standaard-templates/besluit/nieuw/aanvullend-reglement-besluit.ttl [hunk 2]
```diff
@@ -42,7 +42,7 @@
      <li>decreet <a class=\"annotation\" href=\"https://codex.vlaanderen.be/doc/document/1029017\" property=\"eli:cites\" typeof=\"eli:LegalExpression\">over het lokaal bestuur</a> van 22/12/2017</li>
      <li>wet <a class=\"annotation\" href=\"https://codex.vlaanderen.be/doc/document/1009628\" property=\"eli:cites\" typeof=\"eli:LegalExpression\">betreffende de politie over het wegverkeer (wegverkeerswet - Wet van 16 maart 1968)</a></li>
      <li>wegcode - Koninklijk Besluit <a class=\"annotation\" href=\"https://codex.vlaanderen.be/doc/document/1036242\" property=\"eli.cites\" typeof=\"eli:LegalExpression\">van 1 december 1975 houdende algemeen reglement op de politie van het wegverkeer en van het gebruik van de openbare weg.</a></li>
```
Comment (abeforgit) [inline standaard-templates/besluit/nieuw/aanvullend-reglement-besluit.ttl]:
```suggestion
     <li>wegcode - Koninklijk Besluit <a class=\"annotation\" href=\"https://codex.vlaanderen.be/doc/document/1036242\" property=\"eli:cites\" typeof=\"eli:LegalExpression\">van 1 december 1975 houdende algemeen reglement op de politie van het wegverkeer en van het gebruik van de openbare weg.</a></li>
```

**Review by abeforgit — CHANGES_REQUESTED** [review]

**Comment by abeforgit** [comment]
The citation on line 41 is also not correctly annotated. Technically not in scope of this pr, but might as well fix it while we're at it

**Pull request merged**